### DISCUSSION TITLE
Support "Go to definition" for pytest-asyncio fixtures

### DISCUSSION THREAD

**Discussion by meshantz** [microsoft/pylance-release#4031] [Enhancement]
It seems that "Go to Definition" works for a regular pytest fixture. I assume this functionality is provided by the python extension.

It would be great if this would also work for asynchronous fixtures as supported by `pytest-asyncio` https://pypi.org/project/pytest-asyncio/

Example:

```python
async def some_coro():
    return "hello"


@pytest_asyncio.fixture
async def async_example_fails():
    yield await some_coro()


@pytest.fixture
def sync_example_works():
    return "world"


async def test_foo(async_example_fails, sync_example_works):
    assert async_example_fails == "hello"
    assert sync_example_works == "world"
```

Expected behaviour: using the "Go to Definition" (right click menu, or F12) works for both parameters in the test function definition line `async def test_foo(async_example_fails, sync_example_works)`.

This example was produced with:

VSCode 1.75.1  
Python Extension v2023.2.0  
Python 3.11.0  
pytest 7.2.0  
pytest-asyncio 0.20.2

**Comment by rchiodo** [comment]
Thanks for the suggestion. We'll likely turn this into a discussion item to see how popular the request is.

**Comment by daredevil82** [comment]
@rchiodo not sure why this is a discussion vs an issue.  its essentially diverging behavior with pytest and fixtures between sync and async.  That to me indicates a feature gap or bug.
